skywalking/__init__.py

```diff
@@ -43,6 +43,7 @@ class Component(Enum):
     Pyramid = 7009
     Psycopg = 7010
     Celery = 7011
+    Falcon = 7012
```

Comment:
@probeyang can you please add a component id in the main repo and a logo in the UI repo? https://github.com/apache/skywalking/blob/bca62f63fa0c57bf3bb5d8ebde9d6969b55b8429/oap-server/server-bootstrap/src/main/resources/component-libraries.yml#L527-L528

Reply:
ok, i add it.